src/main/scala/org/allenai/scholar/metrics/metadata/GrobidParser.scala

```diff
@@ -0,0 +1,196 @@
+package org.allenai.scholar.metrics.metadata
+
+import java.io.File
+import java.time.Year
+
+import org.allenai.scholar._
+import org.jsoup.Jsoup
+import org.jsoup.nodes.{ TextNode, Document, Element }
+import org.jsoup.select.Elements
+
+import scala.collection.JavaConverters
+import scala.io.Source
+import scala.util.{ Failure, Try }
+
+/** Parser for Grobid XML documents
+  */
+object GrobidParser {
+
+  import Implicits._
+  import StringUtils._
+  import JavaConverters._
+
+  def year(bib: Element): Year =
+    bib.findAttributeValue("monogr>imprint>date[type=published]", "when").extractYear
+
+  def author(e: Element): Option[Author] = Author(
+    lastName = e.findText("persName>surname").toName,
+    firstName = e.findText("persName>forename[type=first]").toName,
+    middleNames = e.select("persName>forename[type=middle]").map(_.text.toName)
+  ).ifDefined
+
+  def extractMetadataAndBib(xmlString: String): MetadataAndBibliography = {
+    val doc = Jsoup.parse(xmlString, "", org.jsoup.parser.Parser.xmlParser())
+    MetadataAndBibliography(metadata = extractMetadata(doc), bibs = extractBibEntriesWithId(doc).map(_._2))
+  }
+
+  private def extractMetadata(doc: Element) = PaperMetadata(
+    title =
+      Title(doc.findText("teiHeader>fileDesc>titleStmt>title")),
+    authors =
+      doc.select("teiHeader>fileDesc>sourceDesc>biblStruct>analytic>author").flatMap(author),
+    year =
+      doc.findAttributeValue("teiHeader>fileDesc>sourceDesc>biblStruct>monogr>imprint>date[type=published]", "when").extractYear,
+    venue =
+      Venue(doc.findText("teiHeader>fileDesc>sourceDesc>biblStruct>monogr>title").toTitle)
+  )
+
+  private def extractBibEntriesWithId(doc: Element) =
+    for {
+      bib <- doc.select("listBibl>biblStruct")
+    } yield {
+      val id = bib.attr("xml:id")
+      val title = bib.findText("analytic>title[type=main]").toTitle match {
+        case "" => Title(bib.findText("monogr>title"))
+        case s => Title(s)
+      }
+      val authors = bib.select("analytic>author").flatMap(author).toList match {
+        case List() => bib.select("monogr>author").flatMap(author).toList
+        case l => l
+      }
+      val venue = Venue(bib.findText("monogr>title"))
```

Comment:
Current behavior of GrobidParser is to only look for `monogr>title` https://github.com/allenai/meta-eval/blob/master/src/main/scala/org/allenai/scholar/metrics/metadata/Parser.scala#L123 

Only the MetaTagger parser looks for "conference", "journal", or "booktitle".  I didn't change its behavior.

Do you think the Grobid parser should be changed to check the alternatives?